distribution/changelog.txt

```diff
@@ -30,6 +30,7 @@
 - Feature: [#14171] [Plugin] Add API for drawing graphics for custom widgets.
 - Feature: [#14171] [Plugin] Add click event to spinners and allow them to be held down.
 - Feature: [#14252] [Plugin] Add API for vehicle g-forces. 
+- Feature: [#13436] Fix MacOS Icon for Big Sur
```

Comment:
```suggestion
- Feature: [#13436] macOS: use new icon that matches Big Sur's style.
```

Also, you need to move this up between 13398 and 13495 - the list is sorted.

Reply:
Oh right, what a dumb mistake.  I've done this before I swear!